xonsh/readline_shell.py

```diff
@@ -453,7 +453,7 @@ def completedefault(self, prefix, line, begidx, endidx):
             multiline_text=prev_text + line,
             cursor_index=len(prev_text) + endidx,
         )
-        rtn_completions = _render_completions(completions, prefix, plen)
+        rtn_completions = _render_completions(completions, line, plen)
```

Comment:
Is it possible to have tests for the rl-completions ? mocking could work or you can use `rl_shell` mock in `test_readline_shell.py` for testing.